src/detect.py

```diff
@@ -0,0 +1,23 @@
+import json
+import os
+
+
+def _get_card_definitions(dir_):
+    return {
+        i.replace(".json", ""): json.load(open(os.path.join(dir_, i)))
+        for i in next(os.walk(dir_))[2]
+        if i.endswith(".json")
+    }
+
+
+script_dir = os.path.dirname(os.path.realpath(__file__))
+
+CARD_DEFINITIONS = _get_card_definitions(os.path.join(script_dir, "card_definitions"))
+
+
+def get_card(record):
+    """Looks at a json record and returns the matching card + card_definition"""
+    for card, card_def in CARD_DEFINITIONS.items():
+        if all(key in record for key in card_def):
+            return card, card_def
```

Comment:
is card_def used downstream? as i see it, right now this function is used for identification purposes so it would be simpler if it returned `company_name` if the card was identified or `None` if match wasn't found.

Reply:
It's true that it's currently not used anywhere else but it's going to be used in the next PR when we re-write the parsing logic to use the card definition returned by this step. I'll leave this in for now.